egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -77,17 +77,17 @@ def AddNoOpLayer(config_lines, name, input):
             'dimension': input['dimension']}
 
 
-
-def AddLdaLayer(config_lines, name, input, lda_file):
+def AddFixedAffineLayer(config_lines, name, input, matrix_file):
```

Comment:
I think this will break the other recipes which use AddLdaLayer function. Add another function AddLdaLayer which will in turn call AddFixedAffineLayer.